dev/devicelab/lib/tasks/perf_tests.dart

```diff
@@ -52,6 +51,15 @@ TaskFunction createComplexLayoutBuildTest() {
   return new BuildTest('${flutterDirectory.path}/dev/benchmarks/complex_layout');
 }
 
+TaskFunction createHelloWorldMemoryTest({ @required DeviceOperatingSystem os }) {
+  return new MemoryTest(
+    '${flutterDirectory.path}/examples/hello_world',
+    null,
```

Comment:
A more idiomatic way to express an optional parameter is to use a named parameter with a default value.